modules/ethereum/src/lib.rs

```diff
@@ -785,19 +963,25 @@ pub(crate) mod tests {
 			parent_hash: &H256,
 		) -> Option<ImportContext<Self::Submitter>> {
 			self.headers.get(parent_hash).map(|parent_header| {
-				let (next_validators_set_start, next_validators) =
-					self.validators_sets.get(&parent_header.next_validators_set_id).unwrap();
+				let validators_set = self
+					.validators_sets
+					.get(&parent_header.next_validators_set_id)
+					.unwrap()
```

Comment:
Another `unwrap`. I just realised that it's all only test code so don't mind the previous comments. Deduplication already logged in https://github.com/paritytech/parity-bridges-common/issues/46